### PR TITLE
add my index

### PR DESCRIPTION
**Pull request by KAYEMBAJONATHAN**
This was my first portfolio web
please check it for review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
     </nav>
 </div>
 <div class="details">
-  <h1>I'm jonathan
+  <h1>I'm Jonathan
```
Comment (Alfredbis29) [inline index.html:24]: - [ ]  I can see that you tried something related to the requirements for this project but I encourage you to follow all requirements for this project, I recommend you to follow this [Requirement for this project](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md)  by reviewing your project I realized that many things don't match the requirements but you tried which is better, keep going by following the requirements for this project then when you will be ready you can submit again :wink:

File: index.html [hunk 2]
```diff
@@ -21,7 +21,7 @@
...
   
     <span>Joe</span></h1>
```
Comment (Alfredbis29) [inline index.html:26]: - [ ] Remember that in this project we imitating the figma design and if we compare your design to the figma one are really different , kindly fix this for your project to look professional , look what can be seen in our browser  :point_down: 
Your Design :point_down: 
part3

Figma design :point_down: 
Screenshot from 2022-12-02 07-44-46

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
Hi @KAYEMBAJONATHAN 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes

_Check the comments under the review._

### OPTIONAL CHANGES :brown_circle: 

- [ ] As required for this project  the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info, kindly put  your container in header tag :wink: 
part1

- [ ] I'm not sure whether you were intended to include a download button in this project; please remove it because it is not part of our project requirements. :wink: 
part4


You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by Alfredbis29 — COMMENTED** [review]
Hi @KAYEMBAJONATHAN , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that linters are not settle , [check this out to install linters in your project](https://github.com/microverseinc/linters-config/tree/master/html-css) 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by KAYEMBAJONATHAN** [comment]
My issue are fixed

**Comment by KAYEMBAJONATHAN** [comment]
fixed

**Pull request closed without merging**